.gitignore

```diff
@@ -15,3 +15,5 @@ node_modules/
 /test_artifacts/
 /coverage/
 yarn-error.log
+/hak/**/*.js
+/scripts/hak/**/*.js
```

Comment:
why would we be expecting js files here?

Reply:
The hak and scripts tsconfigs output alongside the src files. We wouldn't want them in lib anyway and creating another 2 lib directories when most times people will just ts-node them and never need the js files is redundant

Reply:
Ah right, yes.